robot/Cumulus/tests/browser/gift_entry/add_gau_allocations.robot

```diff
@@ -11,7 +11,7 @@ Suite Setup     Run keywords
 Suite Teardown  Run keywords
 ...             API Modify Allocations Setting  ${NS}Default_Allocations_Enabled__c=false   ${NS}Default__c=None
 ...     AND     Query And Store Records To Delete    ${NS}DataImport__c   ${NS}Account1_Name__c=${ACCOUNT_NAME}
-...     AND     Query And Store Records To Delete    ${NS}DataImport__c   ${NS}Contact1_Firstname__c=${FIRST_NAME}  Contact1_Lastname__c=${LAST_NAME}
+...     AND     Query And Store Records To Delete    ${NS}DataImport__c   ${NS}Contact1_Firstname__c=${FIRST_NAME}  ${NS}Contact1_Lastname__c=${LAST_NAME}
```

Comment:
I should know this: when does Robot apply the `${NS}` injection? This field is unmanaged, isn't it?

Reply:
@skristem do you happen to know why this is?

Reply:
@davidmreed this field is managed. We're expecting this test to work as part of the `regression_org` flow. Since the field is packaged, we need the namespace.

Reply:
🤦 I thought it was a different field that wasn't managed. Apologies for the confusion.